orchagent/dash/dashorch.cpp

```diff
@@ -660,6 +756,11 @@ bool DashOrch::addEni(const string& eni, EniEntry &entry)
     }
     eni_entries_[eni] = entry;
 
+    if (entry.metadata.has_trusted_vnis())
+    {
+        addEniTrustedVnis(entry);
```

Comment:
hi Lawrence, what if its an update to change ENI trusted VNI? Can we remove the deleted vnis first and then add the new ones to ensure we do not exceed hardware limits.

Reply:
will add this